include/librealsense2/h/rs_internal.h

```diff
@@ -483,13 +483,23 @@ const char* rs2_get_fw_log_parsed_message(rs2_firmware_log_parsed_message* fw_lo
 const char* rs2_get_fw_log_parsed_file_name(rs2_firmware_log_parsed_message* fw_log_parsed_msg, rs2_error** error);
```

Comment:
I don't see the XML pointer file we spoke about, isn't this in this PR?

Reply:
The definition XML is not kept in the code base. This function outputs the name of the file from which the log entry was registered.